unliftio/unliftio.cabal

```diff
@@ -22,6 +22,11 @@ extra-source-files:
     README.md
     ChangeLog.md
 
+flag old-glibc
+  description: compile with post-2016 glibc, allowing atomic file writes
```

Comment:
Looks like the logic is reversed here. This flag seems to enable the _old_ glibc.